### ISSUE TITLE
config.yaml Error: "has invalid keys: description"

### ISSUE THREAD

**Issue by tuxthepenguin84** [OliveTin/OliveTin#85]
**Describe the bug**
When I have the description field populated in config.yaml I get an error stating:
```
level="error" msg="Config unmarshal error 2 error(s) decoding:\n\n* 'actions[1].Arguments[0]' has invalid keys: description\n* 'actions[1].Arguments[1]' has invalid keys: description"
```

**To Reproduce**
1. Using latest docker image
2. Use example config.yaml (https://github.com/OliveTin/OliveTin/blob/main/config.yaml) and start container (or any config.yaml with description populated)
3. View docker logs and should see the error above

**Expected behavior**
Container should start properly with description field populated. If you remove all the description fields the containers start.

**Screenshots**
```
Attaching to olivetin-home
olivetin-home  | level="info" msg="OliveTin initializing" commit="434e998" date="2022-10-19T14:14:13Z" version="2022-10-19"
olivetin-home  | level="debug" msg="Value of -configdir flag" value="."
olivetin-home  | level="error" msg="Config unmarshal error 2 error(s) decoding:\n\n* 'actions[1].Arguments[0]' has invalid keys: description\n* 'actions[1].Arguments[1]' has invalid keys: description"
olivetin-home exited with code 1
olivetin-home exited with code 1
olivetin-home exited with code 1
olivetin-home exited with code 1
```

**Additional context**
This is an issue I've found using the latest docker image, I have not done any other testing outside of that.

**Comment by tuxthepenguin84** [comment]
Appears this may be functionality in a future release: https://github.com/OliveTin/OliveTin/issues/61

**Comment by jamesread** [comment]
Hey @tuxthepenguin84 , sorry that you had this issue. Yes, I think I updated the config example too early. This should be fixed with the recent 2022.11.11 release, but I need to check.

**Comment by jamesread** [comment]
I've tested this against 2022.11.14, and indeed the example config works. Closing this because it was a temporary issue, but please feel free to re-open / comment if I missed something.
